### PR TITLE
Small changes for Beatnik 1.0 release

### PR DESCRIPTION
**Pull request by patrickb314**
Minor changes to beatnik spack specification for builing on CORAL systems with Spectrum MPI and to add a commit hash for the new 1.0 release. Also a minor change to silo removing an unneeded library search path addition.

### CODE REVIEW

**Review by JStewart28 — APPROVED** [review]
Everything looks good

**Review by JStewart28 — APPROVED** [review]

**Comment by JStewart28** [comment]
Everything looks good

File: var/spack/repos/builtin/packages/beatnik/package.py [hunk 1]
```diff
@@ -14,8 +14,8 @@ class Beatnik(CMakePackage, CudaPackage, ROCmPackage):
 
     maintainers("patrickb314", "JStewart28")
 
-    # Add proper versions and checksums here. Will add 1.0 when a proper SHA is available
-    # version("1.0", sha256="XXX")
+    # Add proper versions and checksums here.
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/beatnik/package.py]:
```suggestion
```

File: var/spack/repos/builtin/packages/silo/package.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def flag_handler(self, name, flags):
             if "+hdf5" in spec:
                 if spec["hdf5"].satisfies("~shared"):
                     flags.append("-ldl")
-            flags.append(spec["readline"].libs.search_flags)
+            # flags.append(spec["readline"].libs.search_flags)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/silo/package.py]: Please, don't leave commented out code. Either remove or modify :slightly_smiling_face:

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/silo/package.py (hunk 2 above)
Comment (patrickb314) [reply var/spack/repos/builtin/packages/silo/package.py]: Fixed, thanks.

**Review by alalazo — APPROVED** [review]

**Pull request merged**